packages/mobile/src/account/Invite.tsx

```diff
@@ -152,9 +152,7 @@ class Invite extends React.Component<Props, State> {
 
   render() {
     return (
-      // Intentionally not using SafeAreaView here as RecipientPicker
-      // needs fullscreen rendering
-      <View style={style.container}>
+      <SafeAreaView style={style.container}>
```

Comment:
Looking at the comment, which was here previously I assume: "RecipientPicker DOES NOT need fullscreen rendering anymore". Is this correct?

Reply:
Yes!